include/qengine_cpu.hpp

```diff
@@ -15,11 +15,10 @@
 #include <algorithm>
```

Comment:
Not that it matters much, but duplicate includes with qengine.hpp

Reply:
Fixed.